Formula/l/llvm.rb

```diff
@@ -445,8 +493,37 @@ def install
     end
   end
 
+  # We use the extra layer of indirection in `arch` because the FormulaAudit/OnSystemConditionals
+  # doesn't want to let us use `Hardware::CPU.arch` outside of `install` or `post_install` blocks.
+  def write_config_files(macos_version, kernel_version, arch)
+    clang_config_file_dir.mkpath
+
+    drivers = ["clang", "clang++", "clang-cpp"]
+    arches = Set.new([:arm64, :x86_64])
+    arches << arch
+
+    arches.each do |arch|
+      drivers.each do |driver|
+        (clang_config_file_dir/"#{arch}-apple-darwin#{kernel_version}-#{driver}.cfg").atomic_write <<~CONFIG
+          --sysroot=#{MacOS::CLT::PKG_PATH}/SDKs/MacOSX#{macos_version}.sdk
+        CONFIG
```

Comment:
Given that we hardcode the CLT here, this won't work at all for Xcode-only installs. Wonder if it's just worth dropping the `pour_bottle?` check and then let users install the CLT if they need to use `clang`.